CompoundV3-Bots/Transfers-Monitor/src/utils.ts

```diff
@@ -0,0 +1,12 @@
+export interface NetworkData {
+  cometAddresses: string[];
+  baseTokens: string[];
```

Comment:
As mentioned, probably better like this:
```suggestion
  cometContracts: { address: string; baseToken: string }[];
```